internal/kldtx/txnprocessor.go

```diff
@@ -406,7 +407,7 @@ func (p *txnProcessor) waitForCompletion(inflight *inflightTxn, initialWaitDelay
 	var elapsed time.Duration
 	for !isMined && !timedOut {
 
-		if isMined, err = inflight.tx.GetTXReceipt(inflight.txnContext.Context(), p.rpc); err != nil {
+		if isMined, err = inflight.tx.GetTXReceipt(context.Background(), p.rpc); err != nil {
```

Comment:
I think rather than changing the context for this one RPC call, it's potentially safer to update the creation of the context.
I believe you saw this because you're using the `webhooksDirect` implementation (rather than Kafka), and so probably it's better to swap to `context.Background()` in this `msgContext`:
https://github.com/kaleido-io/ethconnect/blob/3d441aeceb8be565fd38e044f197e82e6dd18237/internal/kldrest/webhooksdirect.go#L145

Reply:
That makes sense! I will update the PR.